pyfolio/utils.py

```diff
@@ -165,7 +165,13 @@ def get_returns_cached(filepath, update_func, latest_dt, **kwargs):
             raise
         update_cache = True
     else:
-        if pd.Timestamp(mtime, unit='s') < _1_bday_ago():
```

Comment:
It doesn't appear so. It's still being used here:

- `default_returns_fun`
- `load_portfolio_risk_factors`
- `get_treasury_yield`